nbcparser/pkg/parser/helper.go

```diff
@@ -546,7 +546,7 @@ func getGpuImageSha(vmSize string) string {
 }
 
 func getGpuDriverVersion(vmSize string) string {
-	return common.GetGPUDriverVersion(vmSize)
+	return common.GetNewGPUDriverVersion(vmSize)
```

Comment:
could we just call `common.GetNewGPUDriverVersion(vmSize)` directly instead of defining this `getGPUDriverVersion` helper? seems a bit redundant to me